cypress/integration/inventory/assing-tags-to-an-instance-record.spec.js

```diff
@@ -0,0 +1,54 @@
+import uuid from 'uuid';
+import TestTypes from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import getRandomPostfix from '../../support/utils/stringTools';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+describe('ui-inventory: Assign tags to an Instance record', () => {
+  const instanceTitle = `autoTestInstanceTitle.${getRandomPostfix()}`;
+  const tag = {
+    id: uuid(),
+    description: 'autotag',
+    label: 'autotag'
```

Comment:
Leave a comment please why you don't use generate random string
If you have a jira issue with question to PO about tag length limit, leave it here pls